deps/node-inspect/lib/internal/inspect_repl.js

```diff
@@ -86,7 +86,9 @@ function extractFunctionName(description) {
 
 const NATIVES = process.binding('natives');
 function isNativeUrl(url) {
-  return url.replace('.js', '') in NATIVES || url === 'bootstrap_node.js';
+  return url.replace('.js', '') in NATIVES ||
```

Comment:
Should a PR be opened against https://github.com/nodejs/node-inspect before this lands so it doesn't get forgotten?

Also, strictly speaking, this change should either be in the same or preceding commit to the actual filename change (rather than after it), as otherwise node-inspect is broken.

(I just skimmed the PR so I could be totally off base.)

Reply:
@apapirovski I just realized this is not necessary since

```
> Object.keys(process.binding('natives'))
[ 'internal/bootstrap/loaders',
  'internal/bootstrap/node',
  ...
```